### PR TITLE
Lead connections agg tables

### PR DESCRIPTION
**Pull request by kailamjeter**
## Description

Addresses: [Complete data transformation](https://app.shortcut.com/blueconduit/story/5678/complete-data-transformation)

Add precomputed tables at state, county, zip level for lead_connections (water_systems table). Modify lead_connections_function_source to point to this state table.

### New

- Table defs + inserts

### Changed

- Lead connections function source

### Removed

- N/A

## Testing and Reviewing

run schema.sql, run npm run serve to view data in FE

### CODE REVIEW

File: cdk/src/open-data-platform/data-plane/schema/schema.sql [hunk 1]
```diff
@@ -152,22 +166,11 @@ CREATE TRIGGER update_last_update_timestamp
...
 
-CREATE INDEX IF NOT EXISTS geom_index ON counties USING GIST (geom);
-CREATE UNIQUE INDEX IF NOT EXISTS zipcode_index ON zipcodes (zipcode);
+-- Demographic aggregation tables.
 
--- Precomputed table is required to ensure acceptable latency for the
--- tileserver.
 CREATE TABLE IF NOT EXISTS state_demographics
 (
     census_geo_id         varchar(255) NOT NULL,
```
Comment (justinebreuch) [inline cdk/src/open-data-platform/data-plane/schema/schema.sql:176]: I know this was me but can you change black and white population to reals?

File: cdk/src/open-data-platform/data-plane/schema/schema.sql [hunk 2]
```diff
@@ -216,8 +215,49 @@ CREATE TABLE IF NOT EXISTS zipcode_demographics
...
 
--- Pre-computed demographic data by state
--- Only to be used by the function source
+-- Lead connections aggregation tables.
+
+CREATE TABLE IF NOT EXISTS state_lead_connections
+(
+    census_geo_id             varchar(255) NOT NULL,
+    name                      varchar(255) NOT NULL,
+    lead_connections_count    float,
```
Comment (justinebreuch) [inline cdk/src/open-data-platform/data-plane/schema/schema.sql:224]: Can these counts be reals?

Comment (justinebreuch) [reply cdk/src/open-data-platform/data-plane/schema/schema.sql:224]: Here and below

**Review by justinebreuch — APPROVED** [review]

File: cdk/src/open-data-platform/data-plane/schema/schema.sql (hunk 2 above)
Comment (kailamjeter) [reply cdk/src/open-data-platform/data-plane/schema/schema.sql:224]: Done

**Pull request merged**